### PR TITLE
Create Vose-Alias-Method conda recipe

### PR DESCRIPTION
**Pull request by asmith26**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
  - "I created the meta.yaml with `conda skeleton pypi Vose-Alias-Method` so I think has been created correctly(?)."
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/Vose-Alias-Method/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  imports:
+    - vose_sampler
+  commands:
+    - vose-sampler --help
+
+about:
+  home: "https://github.com/asmith26/Vose-Alias-Method.git"
+  license: "Apache-2.0"
+  license_family: "APACHE"
+  license_file: ""
```
Comment (marcelotrevisani) [inline recipes/Vose-Alias-Method/meta.yaml]: Could you please add the ``license_file``?

File: recipes/Vose-Alias-Method/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "6ddc19008b917cbe46f3b2251ea0a5588f2261ad7b6f4f827285b47f9e099040"
+
+build:
+  number: 0
+  entry_points:
+    - vose-sampler=vose_sampler.vose_sampler:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/Vose-Alias-Method/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/Vose-Alias-Method/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "6ddc19008b917cbe46f3b2251ea0a5588f2261ad7b6f4f827285b47f9e099040"
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/Vose-Alias-Method/meta.yaml]: Could it be ``noarch``?
```suggestion
  number: 0
  noarch: python
```

File: recipes/Vose-Alias-Method/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    - vose-sampler --help
+
+about:
+  home: "https://github.com/asmith26/Vose-Alias-Method.git"
+  license: "Apache-2.0"
+  license_family: "APACHE"
+  license_file: ""
+  summary: "Python implementation of Vose's alias method, an efficient algorithm for sampling from a discrete probability distribution."
+  doc_url: ""
+  dev_url: ""
```
Comment (marcelotrevisani) [inline recipes/Vose-Alias-Method/meta.yaml]: Could you please delete or fill the ``doc_url`` and ``dev_url`` ?

**Comment by asmith26** [comment]
Hi @marcelotrevisani 

Many thanks for all your feedback. This is my first push to conda-forge so all feedback is greatly appreciated.

I've pushed a change to this PR which I think fixes all your comments.

Many thanks again!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**